### PR TITLE
Added overview page

### PR DESCRIPTION
**Pull request by dancingcactus**
Adds a small overview page that explains what Alloy is. 

## Checklist:
- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: get-started/overview.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+description: >-
+  An overview of what Alloy is and how it can be used. 
+---
+
+{% hint style="warning" %}
+This documentation is for a library and a service that is in Alpha and should not be used for production use-cases.
```
Comment (Aaronius) [inline get-started/overview.md]: `use-cases` -> `use cases`

File: get-started/overview.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+description: >-
+  An overview of what Alloy is and how it can be used. 
+---
+
+{% hint style="warning" %}
+This documentation is for a library and a service that is in Alpha and should not be used for production use-cases. 
+{% endhint %}
+
+# What is Alloy
+Alloy is a client side javascript library that allows customers of the Adobe Experience Cloud to interact with the various services in teh Experience Cloud. It is meant to replace the following SDKs.
```
Comment (Aaronius) [inline get-started/overview.md]: `client side javascript` -> `client-side JavaScript`
`teh` -> `the`

File: get-started/overview.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+{% endhint %}
+
+# What is Alloy
+Alloy is a client side javascript library that allows customers of the Adobe Experience Cloud to interact with the various services in teh Experience Cloud. It is meant to replace the following SDKs. 
+- Visitor.js
+- AppMeasurment.js
+- AT.js
+- DIL.js
+
+This is a complete re-write not just a wrapper around the exisitng libraries. It's purpose is to end challenges with tags firing in the right order, inconsistency with library versioning challenges and better dependency management. It is a new way to implement the Experience Cloud.
```
Comment (Aaronius) [inline get-started/overview.md]: `This is a complete re-write not` -> `This is a complete rewrite--not` (or you can change the sentence structure to something like `This is not just a wrapper around the existing libraries; it is a complete rewrite.`
`exisitng` -> `existing`
`It's` -> `Its`

File: get-started/overview.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+# What is Alloy
+Alloy is a client side javascript library that allows customers of the Adobe Experience Cloud to interact with the various services in teh Experience Cloud. It is meant to replace the following SDKs. 
+- Visitor.js
+- AppMeasurment.js
+- AT.js
+- DIL.js
+
+This is a complete re-write not just a wrapper around the exisitng libraries. It's purpose is to end challenges with tags firing in the right order, inconsistency with library versioning challenges and better dependency management. It is a new way to implement the Experience Cloud. 
+
+In addtion to a new library, there will be a new end point that streamlines the HTTP requests to Adobe solutions. Before where there used to be a blocking call to the visitor ID service, then a call to Adobe Target, Then a call to Adobe Audience Manager and finally a call to Adobe Analytics. This new world will have a call that can retrieve an ID, fetch a Target experience, send data to Audience Manager and pass the data into the Adobe Experience Platform.
```
Comment (Aaronius) [inline get-started/overview.md]: `addtion` -> `addition`
`end point` -> `endpoint`
`Before where there used to be a blocking call to the visitor ID service, then a call to Adobe Target, Then a call to Adobe Audience Manager and finally a call to Adobe Analytics.` -> `Before, a blocking call would be made to the visitor ID service, then another to Adobe Target, then another to Adobe Audience Manager, and finally another to Adobe Analytics.`
`This new world will have a call` -> `This new world will have a single call` (to emphasize that it's a single call that does all the things)

Comment (dancingcactus) [reply get-started/overview.md]: Perfect Thanks!

File: get-started/overview.md (hunk 3 above)
Comment (dancingcactus) [reply get-started/overview.md]: 👍

**Pull request merged**